### PR TITLE
mcs: update store stats

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5839.

### What is changed and how does it work?

Make scheduler server can update the store statistics.

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

File: pkg/mcs/scheduling/server/cluster.go [hunk 1]
```diff
@@ -348,3 +348,28 @@ func (c *Cluster) HandleStoreHeartbeat(heartbeat *schedulingpb.StoreHeartbeatReq
 	c.hotStat.CheckReadAsync(statistics.NewCollectUnReportedPeerTask(storeID, regions, interval))
 	return nil
 }
+
+// RunUpdateStoreStats updates store stats periodically.
+func (c *Cluster) RunUpdateStoreStats() {
+	defer logutil.LogPanic()
+
+	ticker := time.NewTicker(9 * time.Millisecond)
```
Comment (lhy1024) [inline pkg/mcs/scheduling/server/cluster.go:356]: 9 * time.Millisecond?

Comment (rleungx) [reply pkg/mcs/scheduling/server/cluster.go:356]: keep the same as previous and less than the interval of store heartbeat.

Comment (rleungx) [reply pkg/mcs/scheduling/server/cluster.go:356]: Keep the same as previous.

**Review by JmPotato — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**